src/Message/AuthorizeRequest.php

```diff
@@ -56,12 +66,34 @@ public function getData()
                 "OrderId"     => $this->getTransactionId(),
                 "Description" => $this->getDescription(),
             ],
-            "ReturnUrls"    => [
-                "Success" => $this->getReturnUrl(),
-                "Fail"    => $this->getCancelUrl(),
+            "ReturnUrl"    => [
+                "Url" => $this->getReturnUrl(),
```

Comment:
@nidjo17 changing the structure of the returned values directly will break the existing implementations. The new values should be added to the existing ones or abstracted in a way so that it's backward compatible

Reply:
The current solution was not compatible with the newer version of Saferpay. Saferpay no longer accepts the array structure with the keys "Success" and "Cancel" (they’ve changed the request body) in the latest versions. I understand that this update could cause issues and might be considered a breaking change, so I suggest versioning it as v2.

I made these changes based on my own project needs, and they have been tested and are running in production on my end. If there's interest in merging the PR, I can adapt the changes.